sphere-common/src/main/java/io/sphere/sdk/queries/IsGreaterThanOrEqualsQueryPredicate.java

```diff
@@ -0,0 +1,13 @@
+package io.sphere.sdk.queries;
+
+class IsGreaterThanOrEqualsQueryPredicate<T, V, M> extends ComparisonQueryPredicate<T, V, M> {
+
+    public IsGreaterThanOrEqualsQueryPredicate(final QueryModel<M> queryModel, final V value) {
```

Comment:
Ok, I put https://github.com/sphereio/sphere-jvm-sdk/issues/407 into the current milestone.